validation/taylor_green_vortex_2d/validation_taylor_green_vortex_2d.jl

```diff
@@ -0,0 +1,84 @@
+using TrixiParticles
+
+# ==========================================================================================
+# ==== Resolution
+particle_spacings = [0.02, 0.01, 0.005]
+
+# ==========================================================================================
+# ==== Experiment Setup
+tspan = (0.0, 5.0)
+reynolds_number = 100.0
+
+function compute_L1v_error(v, u, t, system)
+    v_analytical_avg = 0.0
+    L1v = 0.0
```

Comment:
```suggestion
    v_analytical_avg = 0.0
    v_avg = 0.0
```
Make clear that it's the same but numerical vs analytical solution?